docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -66,19 +67,36 @@ If you are using {agent} with link:{serverless-docs}[{serverless-full}], note th
 ====
 
 [discrete]
-== Minimum Requirements
+[[elastic-agent-installation-minimum-requirements]]
+== Minimum requirements  
+
+The following are the minimum system requirements for installing {agent}.
+// lint ignore mem
+[discrete]
+=== CPU and RSS memory size
 
 // lint ignore 2vcpu 1gb
 Minimum requirements have been determined by running the {agent} on a GCP `e2-micro` instance (2vCPU/1GB).
 The {agent} used the default policy, running the system integration and self-monitoring.
-During upgrades, double the disk space is required to store the new {agent} binary. After the upgrade completes, the original {agent} is removed from disk to free up the space.  
 
-// lint ignore mem
 |===
 | **CPU** | Under 2% total, including all monitoring processes
-| **Disk** | 1.7 GB
-| **RSS Mem Size** | 400 MB
+| **RSS memory size** | 400 MB
 |===
 Adding integrations will increase the memory used by the agent and its processes.
 
+[discrete]
+=== Size on disk
+
+The disk requirements for {agent} vary by operating system and {stack} version. With version 8.14 we have significantly reduced the size of the {agent} binary. Further reductions will be made in future releases.
```

Comment:
Thanks @shainaraskas. I've changed it to `Further reductions *are planned to be made* in future releases.` It's important that this is included.

For the "starting with" part, I've left as is because there may have been some size adjustment work done in previous releases.